src/FocusTrap.ts

```diff
@@ -54,13 +55,25 @@ export const FocusTrap = defineComponent({
 
   setup(props, { slots, emit }) {
     let trap: FocusTrapI | null
-    const el = ref<HTMLElement | null>(null)
+    const el = ref<HTMLElement | ComponentPublicInstance | null>(null)
 
     const ensureTrap = () => {
       if (trap) {
         return
       }
 
+      if (el === null || el?.value === null) {
```

Comment:
This should be done in dev mode only

```suggestion
      if (__DEV__ && !el.value) {
```

Reply:
`__DEV__` doesn't work here. Not sure about the env setup for the project, but it complains in the browser console that `__DEV__` is not defined, at least when I'm testing it with `yarn dev`.

If `ensureTrap` is not supposed to throw this error, can definitely remove it, but how shall we handle the case where `el.value` is undefined?

Reply:
Ah, I will have to give it a look and add it. We can just do `process.env.NODE_ENV !== 'production'`
It should only throw in DEV as this error should be fixe during dev and never happen in production